next.config.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
     config.resolve.alias['components'] = path.join(__dirname, 'src/components');
     config.resolve.alias['lib'] = path.join(__dirname, 'src/lib');
     config.resolve.alias['pages'] = path.join(__dirname, 'src/pages');
+    config.resolve.alias['@redux'] = path.join(__dirname, 'src/redux');
```

Comment:
Can you please explain this @ prefix for me?

Reply:
Next's package/import resolution isn't the best by default, and attempting to alias the redux folder without it causes errors as it can't figure out if you're trying to import from the package `redux` or the folder of the same name.

Since my original PR a few months ago this is actually a pattern we've started using everywhere to distinguish local vs package imports.

Reply:
So @ always means local? Should we change the others too for consistency?

Reply:
Yep, I was planning to change imports on the other ones after this PR.